### PR TITLE
docs: typo fixes

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -I ../mantid-word-whitelist.txt` where whitelist consists of:  
```
alloced
allws
ang
ans
arithmetics
dum
dur
iff
isnt
ith
lod
mut
nd
originaly
sav
splitted
te
tim
uint
```

### CODE REVIEW

File: docs/source/algorithms/GeneratePeaks-v1.rst [hunk 1]
```diff
@@ -116,7 +116,7 @@ Generate peaks from a TableWorkspace
 .. testcleanup:: GeneratePeakFromTable
 
   DeleteWorkspace(Workspace=outws)
-  DeleteWorkspace(Workspace="focussed")
+  DeleteWorkspace(Workspace="focused")
```
Comment (rosswhitfield) [inline docs/source/algorithms/GeneratePeaks-v1.rst]: This change is causing a doc test to fail. You don't need to *fix* any of the **focussed** words as this is correct for British English.

**Review by rosswhitfield — CHANGES_REQUESTED** [review]
Nice! Thanks

File: docs/source/algorithms/ConvertToReflectometryQ-v1.rst [hunk 2]
```diff
@@ -146,14 +146,14 @@ Before the ConvertToReflectometryQ algorithm is executed in the usage example ab
 .. figure:: /images/SignedThetaVSlam_plot.png
    :alt: plot of Signed theta vs lambda. 
     
-Patch Plot with Dumped Vertexes
+Patch Plot with Dumped Vertices
```
[the remark this answers is not in the document]
Comment (luzpaz) [reply docs/source/algorithms/ConvertToReflectometryQ-v1.rst]: Reverted in 7b9f043

File: docs/source/algorithms/GeneratePeaks-v1.rst (hunk 1 above)
Comment (luzpaz) [reply docs/source/algorithms/GeneratePeaks-v1.rst]: Reverted in 7b9f043

**Review by rosswhitfield — APPROVED** [review]

**Pull request merged**